Source/gendung.cpp

```diff
@@ -597,4 +597,21 @@ void FindTransparencyValues(int i, int j, int x, int y, int d, uint8_t floorID)
 	FindTransparencyValues(i + 1, j + 1, x + 2, y + 2, 8, floorID);
 }

```

Comment:
`FindTransparencyValues` can be moved into the anonymous namespace at the start of gendung.cpp now that it's only called from one version of `FloodTransparencyValues` :D